qiskit_ibm_runtime/utils/backend_converter.py

```diff
@@ -12,192 +12,307 @@
 
 """Converters for migration from IBM Quantum BackendV1 to BackendV2."""
 
+from __future__ import annotations
+
+import logging
+import warnings
 from typing import Any, Dict, List
 
-from qiskit.transpiler.target import Target, InstructionProperties
-from qiskit.utils.units import apply_prefix
-from qiskit.circuit.library.standard_gates import (
-    IGate,
-    SXGate,
-    XGate,
-    CXGate,
-    RZGate,
-    ECRGate,
-    CZGate,
+from qiskit.circuit.controlflow import (
+    CONTROL_FLOW_OP_NAMES,
+    ForLoopOp,
+    IfElseOp,
+    SwitchCaseOp,
+    WhileLoopOp,
 )
-from qiskit.circuit.controlflow import CONTROL_FLOW_OP_NAMES
-from qiskit.circuit import IfElseOp, WhileLoopOp, ForLoopOp, SwitchCaseOp
-from qiskit.circuit.parameter import Parameter
-from qiskit.circuit.delay import Delay
 from qiskit.circuit.gate import Gate
-from qiskit.circuit.measure import Measure
-from qiskit.circuit.reset import Reset
-from qiskit.providers.models import (
-    BackendConfiguration,
-    BackendProperties,
-    PulseDefaults,
-)
+from qiskit.circuit.library.standard_gates import get_standard_gate_name_mapping
+from qiskit.circuit.parameter import Parameter
+from qiskit.providers.backend import QubitProperties
+from qiskit.providers.exceptions import BackendPropertyError
+from qiskit.providers.models.backendconfiguration import BackendConfiguration
+from qiskit.providers.models.backendproperties import BackendProperties
+from qiskit.providers.models.pulsedefaults import PulseDefaults
+from qiskit.transpiler.target import InstructionProperties, Target
 
-from ..ibm_qubit_properties import IBMQubitProperties
```

Comment:
qiskit-ibm-runtime has its own qubit property class. I'm wondering whether I can replace it with Qiskit's qubit properties or not.

Reply:
I believe this is fine, `qubit_props_list_from_props` is a helper function only used in `convert_to_target`. It's not a breaking change